### ISSUE TITLE
404

### ISSUE THREAD

**Issue by swambold6** [stillmuseum/patricia#165]
@halfempty Getting a 404 error trying to get to this page:

https://clyffordstillmuseum.org/photography-policy/

https://clyffordstillmuseum.org/wp-admin/post.php?post=7141&action=edit

Can't figure it out.

**Comment by swambold6** [comment]
@halfempty Same problem with this page:

https://clyffordstillmuseum.org/photography-application-received/

https://clyffordstillmuseum.org/wp-admin/post.php?post=7172&action=edit

What gives?

**Comment by halfempty** [comment]
@swambold6 I'm not sure. I was able to do a work around.

I'll leave this open, and will you alert me if it keeps happening?

**Comment by swambold6** [comment]
Absolutely. Happened three or four times today. The first time I thought it
was because of a / in the page title, so I played around with the title /
slug until it worked. Basically any page I published today I had this
problem.

**Comment by halfempty** [comment]
@swambold6 I'm worried this is a bug related to the "Custom Permalinks" plugin we use. This will take more investigation.

The work around I used was to modify the path at the top of the Edit screen so that it differs slightly from the "slug" (which comes in a box lower down the page. this box can be shown via a toggle under Screen Options). Maybe that can be used temporarily while we diagnose?

**Comment by swambold6** [comment]
@halfempty Would like to pick back up with this issue today. Do you have specific instructions for me for specific pages? Not sure what the next steps on this are... ?
